docs/manual/index-checker.tex

```diff
@@ -156,12 +156,16 @@ \section{Lower bounds\label{index-lowerbound}}
 
 The Index Checker issues an error when
 a sequence is indexed by an integer that might be negative.
-It uses a type system (Figure~\ref{fig-index-int-types}) with the following
+The Lower Bound Checker uses a type system (Figure~\ref{fig-index-int-types}) with the following
 qualifiers:
 
 \begin{description}
 \item[\refqualclass{checker/index/qual}{Positive}]
   The value is 1 or greater, so it is not too low to be used as an index.
+  Note that this annotation is trusted by the Constant Value Checker,
```

Comment:
I'm a bit concerned that such behavior would confuse users:  "I see `@Positive` right here, and in other code it is taken account of, but not in this code."  That's why I think a warning (which users could suppress) might be better.